src/components/TextField/TextField.tsx

```diff
@@ -158,10 +158,15 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
         if (inputRef.current) {
           inputRef.current.focus();
         }
-        clearValue();
+        inputRef.current.value = "";
```

Comment:
is that intentional? it wasn't there before, I guess `clearValue` should handle it by itself on the returned `inputValue`?

Reply:
Yeah, without it it was sending `target=inputRef.current` was containing not empty value, but the previous one

Reply:
So what if you were to move the logic of contructing the event sent back on the debounce hook?